core/scripts/ocr2vrf/util.go

```diff
@@ -220,6 +220,30 @@ func setProducer(e helpers.Environment, vrfCoordinatorAddr, vrfBeaconAddr string
 	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
 }
 
+func createSubscription(e helpers.Environment, vrfCoordinatorAddr string) {
```

Comment:
Should we call this as part of the `setup-ocr2vrf` command? Can create a subscription then add the consumer to the subscription as part of that too (using the `addConsumer` you defined below)

Reply:
yeah makes sense! added

Reply:
also added auto funding subscription